projects/1.21/assets/butchercraft/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,321 @@
+  "block.butchercraft.cow_skull_head": "牛头骨",
+  "block.butchercraft.goat_block": "生山羊肉块",
+  "block.butchercraft.goat_head": "山羊头",
+  "block.butchercraft.goat_hide_carpet": "山羊皮",
+  "block.butchercraft.goat_skull_head": "山羊头骨",
+  "block.butchercraft.grinder": "绞肉机",
+  "block.butchercraft.meat_hook": "挂肉钩",
+  "block.butchercraft.mutton_block": "生羊肉块",
+  "block.butchercraft.pig_head": "猪头",
+  "block.butchercraft.pig_hide_carpet": "猪皮",
+  "block.butchercraft.pig_skull_head": "猪头骨",
+  "block.butchercraft.pork_block": "生猪肉块",
+  "block.butchercraft.rabbit_black_head": "黑色兔头",
+  "block.butchercraft.rabbit_block": "兔肉块",
+  "block.butchercraft.rabbit_brown_head": "棕色兔头",
+  "block.butchercraft.rabbit_gold_head": "金色兔头",
+  "block.butchercraft.rabbit_salt_head": "胡椒盐色兔头",
+  "block.butchercraft.rabbit_skull_head": "兔头骨",
+  "block.butchercraft.rabbit_splotched_head": "斑点兔头",
+  "block.butchercraft.rabbit_white_head": "白兔头",
+  "block.butchercraft.sheep_head": "绵羊头",
+  "block.butchercraft.sheep_skull_head": "绵羊头骨",
+  "block.butchercraft.taxidermy_black_rabbit_block": "黑色兔子标本",
+  "block.butchercraft.taxidermy_brown_rabbit_block": "棕色兔子标本",
+  "block.butchercraft.taxidermy_chicken_block": "鸡标本",
+  "block.butchercraft.taxidermy_cow_block": "牛标本",
+  "block.butchercraft.taxidermy_goat_block": "山羊标本",
+  "block.butchercraft.taxidermy_gold_rabbit_block": "金色兔子标本",
+  "block.butchercraft.taxidermy_pig_block": "猪标本",
+  "block.butchercraft.taxidermy_salt_rabbit_block": "胡椒盐色兔子标本",
+  "block.butchercraft.taxidermy_sheep_block": "山羊标本",
+  "block.butchercraft.taxidermy_splotched_rabbit_block": "斑点兔子标本",
+  "block.butchercraft.taxidermy_white_rabbit_block": "白色兔子标本",
+  "block.butchercraft.trimmed_barn_wood_block": "染边畜棚木板",
+  "butchercraft.advancement.apron.desc": "制作屠宰围裙来避免染上血迹",
+  "butchercraft.advancement.apron.name": "热爱烹饪！",
+  "butchercraft.advancement.barn_wood.desc": "制作一些畜棚木板",
+  "butchercraft.advancement.barn_wood.name": "原来还可以这么用",
+  "butchercraft.advancement.blood.desc": "获取一桶鲜血",
+  "butchercraft.advancement.blood.name": "牲畜们的活阎王！",
+  "butchercraft.advancement.bloody.desc": "获得血腥效果",
+  "butchercraft.advancement.bloody.name": "满手都是鲜血",
+  "block.butchercraft.sausage_block": "香肠块",
+  "block.butchercraft.sheep_hide_carpet": "绵羊皮",
+  "butchercraft.advancement.bonesaw.desc": "制作一把骨锯",
+  "butchercraft.advancement.bonesaw.name": "骷髅的噩梦",
+  "butchercraft.advancement.boots.desc": "制作橡胶靴以防止走路产生血液痕迹",
+  "butchercraft.advancement.boots.name": "专为步行设计",
+  "butchercraft.advancement.bunny_equip.desc": "给兔子装上耳朵和尾巴",
+  "butchercraft.advancement.bunny_equip.name": "不包括紧身衣和袜子！",
+  "butchercraft.advancement.butcherblock.desc": "制作一个案板",
+  "butchercraft.advancement.butcherblock.name": "切切切！砍砍砍！",
+  "butchercraft.advancement.butcherknife.desc": "制作一把屠刀",
+  "butchercraft.advancement.butcherknife.name": "屠夫不杀五爪猪",
+  "butchercraft.advancement.cannibalism.desc": "试试去宰了一个人类",
+  "butchercraft.advancement.cannibalism.name": "还有人性吗⋯⋯",

```

```suggestion
  "butchercraft.advancement.cannibalism.name": "还有人性吗……",
```
改省略号，大部分输入法的<kbd>Shift</kbd>+<kbd>6</kbd>，全文通查

---

已修改